kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/admin/UserTaskAdminServiceBase.java

```diff
@@ -68,6 +69,8 @@ public UserTaskAdminServiceBase(UserTaskAdminService userTaskAdminService, KieSe
 
     public void addPotentialOwners(String containerId, Number taskId, boolean removeExisting, String payload, String marshallingType) {
         logger.debug("About to unmarshall payload '{}' to map of task inputs", payload);
+        if (context.getContainer(containerId) == null) throw new DeploymentNotFoundException("Could not find container with ID: " + containerId);
```

Comment:
I love code reviews!  I had planned on doing that after my brute-force way worked and forgot.